instrumentation/github.com/gin-gonic/gin/otelgin/gintrace.go

```diff
@@ -95,7 +95,10 @@ func Middleware(service string, opts ...Option) gin.HandlerFunc {
 			span.SetAttributes(semconv.HTTPStatusCode(status))
 		}
 		if len(c.Errors) > 0 {
-			span.SetAttributes(attribute.String("gin.errors", c.Errors.String()))
+			span.SetStatus(codes.Error, "gin error")
```

Comment:
`gin error` isn't very meaningful.

```suggestion
			span.SetStatus(codes.Error, c.Errors.String())
```

Reply:
Wont this duplicate, in a less useful flattened string from, the information being added as events below?

Reply:
@akats7 We can discuss it here. It should be the same issue here.

Reply:
I recommend using this method to record: `span.SetStatus(codes.Error, c.Errors.String())`

I would like to hear everyone's suggestions.

Reply:
From: https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6346#issuecomment-2498987933

> I personally always find it convenient to have the error message at least partially included directly on the span instead of **only** having the span events. Since span events are sometimes (always?) stored in separate table it could becomes an additional step to view the message depending on the platform you use. 
>
> For example I know a lot of the java instrumentation just adds _error: true_ as an attribute and then creates a span event, and we've had teams complain that the error message has no contextual info

Reply:
Personally, I do not mind duplication.

Reply:
Based on this reason, if I adjust it like this, is it appropriate?

```go
span.SetStatus(codes.Error, c.Errors.String())
```

Reply:
@MrAlias, most of us prefer `span.SetStatus(codes.Error, c.Errors.String())`.

Please create an issue or follow-up PR if you think that this is a wrong decision.